### DISCUSSION TITLE
Sign automerged squashed commits

### DISCUSSION THREAD

**Discussion by donmahallem** [renovatebot/renovate#12423] [Request Help]
While commits are signed by renovate if those pull requests are squash merged by renovate they aren't signed any more.

![Screenshot of merged PRs](https://user-images.githubusercontent.com/4698322/139673204-efd57d5a-17d1-4fbc-9128-71989f6a5e50.png)

Is this is a bug or  a technical limitation of renovate?

**Comment by viceice** [comment]
It depends on the platform. Which are you using?

**Reply by donmahallem** [reply]
Github

**Comment by rarkins** [comment]
My first guess is that this is a limitation of GitHub's. Automerge setting is done via API, so impossible to "sign" the merge. If you set `platformAutomerge` to `false` in your repo config then it should revert back to using Renovate's own merging.
